UPDATING.md

```diff
@@ -1043,6 +1019,12 @@ dag.get_task_instances(session=your_session)
 
 ## Airflow 1.10.3
 
+### New `dag_discovery_safe_mode` config option
+
+If `dag_discovery_safe_mode` is enabled, only check files for DAGs if
+they contain the strings "airflow" and "DAG". For backwards
+compatibility, this option is enabled by default.
+
```

Comment:
This is already in `Airflow 1.10.3` section. Check https://github.com/apache/airflow/pull/7166/files#diff-bb7ee25332d5fdca3ead2f545d72b359R1061-R1065

Reply:
Ah, my mistake. I'll just remove it then.